radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Ownable.java

```diff
@@ -1,10 +1,12 @@
 package com.radixdlt.client.atommodel;
 
+import com.radixdlt.client.atommodel.accounts.RadixAddress;
 import com.radixdlt.client.core.crypto.ECPublicKey;
 
 /**
  * Temporary interface for representing a particle that can be owned and only invalidated by an owner.
  */
 public interface Ownable {
 	ECPublicKey getOwner();
+	RadixAddress getAddress();
```

Comment:
Hmm I think it's weird we have both, we shouldn't need both - can we remove getAddress() and just make `getOwner` return `RadixAddress`